icu4j/main/charset/pom.xml

```diff
@@ -23,7 +22,6 @@
   </scm>
 
   <properties>
-    <icu4j.api.doc.root.dir>${project.basedir}/../..</icu4j.api.doc.root.dir>
     <module-name>charset</module-name>
```

Comment:
What does the `module-name` do that is different from `artifactId`? Do sibling submodules have a custom `module-name`? If not, why does this submodule have it?

Reply:
Buy default the module name is identical to the `<name>`.

By default the module name is the same as the name.
BUT! Using `-` in a module name is invalid.
So `module-name` is only required for artifacts that have invalid names (with `-`)

The module names end up in manifest, so I didn't change that, especially in the two artifacts we publish.

For Java modules see https://www.baeldung.com/java-modularity

Reply:
Confusing but makes sense.